jbpm-case-mgmt/jbpm-case-mgmt-impl/src/test/java/org/jbpm/casemgmt/impl/CaseRuntimeDataServiceImplTest.java

```diff
@@ -46,6 +48,7 @@
 import org.slf4j.Logger;
 import org.slf4j.LoggerFactory;
 
+import static org.jbpm.services.task.wih.util.PeopleAssignmentHelper.ACTOR_ID;
```

Comment:
Currently, this is not used, but it may be as well as other constants from this helper

Reply:
Done